Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataTreeTable.xhtml

```diff
@@ -44,15 +44,21 @@
                  id="metadataTable">
         <p:column>
             <span class="input-wrapper">
-                <!-- field label -->
-                <p:outputLabel value="#{item.label}:" rendered="#{not item.undefined}"/>
-                <p:outputLabel value="#{item.label}: ⚠️" style="background-color: gold;"
-                               title="#{msgs['dataEditor.undefinedKey']}" rendered="#{item.undefined}" />
+                <!-- field label
+                The field ID must match the field's input attribute "#{item.input}", so that the outputLabel can be mapped to the field with the "for" attribute -->
+                
+                <p:outputLabel for="#{item.input eq 'dataTable' ? '' : item.input}"
+                               value="#{item.label}: #{item.required ? ' *' :''}"
+                               rendered="#{not item.undefined}"/>
+                <p:outputLabel value="#{item.label}: ⚠️"
```

Comment:
What is the last character here?

Reply:
` <p:outputLabel value="#{item.label}: ⚠️"
                               style="background-color: gold;"
                               title="#{msgs['dataEditor.undefinedKey']}"
                               rendered="#{item.undefined}"/>`
   
  If an imported metadata is not defined in the ruleset, its label is marked with this character  ` ⚠️`

Reply:
Thanks i thought it was an incorrect character.